README.md

```diff
@@ -21,6 +21,7 @@ library to compile the native extensions.
 ### OSX
 
     xcode-select --install
+    rvm install 1.9
```

Comment:
This adds a dependency on `rvm`, or is that part of OSX now? Isn't there a ruby provided by OSX anymore?